### ISSUE TITLE
Define a RESTful API route for database queries

### ISSUE THREAD

**Issue by signebedi** [signebedi/libreForms#4]
As the title suggests, this would allow users to query database information by hitting routes like `/api/v1/API_TOKEN?query=True&options=test&here=1`.

**Comment by signebedi** [comment]
Should we employ API keys and, if so, what is the best way to implement them? Right now, I am having app.py look for a file called `api_keys`, which is structured with a header line, and then line-breaks to divide keys as below:

```
api_keys
FFvHqtFDl7G9bhWoZ8VHT3M9 
4mlAqqcyzLwQ4IsHKoEiV3ft 
zNItarEPNGCgeIhqEex8OYvJ 
t32HDBcKAAIVBBPbjBADCbCh
```
Nb. this file does not ship with the application and needs to be created.

**Comment by signebedi** [comment]
This feature is implemented preliminarily in 5155802. Future work can be done to define a more robust (albeit complex) backend for managing API keys... an `api_keys` file is probably too basic for enterprise use.
